### DISCUSSION TITLE
Ability to modify duration of every part of the meeting

### DISCUSSION THREAD

**Discussion by Nickyes** [AntonyCorbett/OnlyT#511] [Ideas]
Hello, and first, thanks for all the improvements; They are highly appreciated.

In order to include special parts, the branch has sometime recently indicated that the duration of some of them (which, according to the instructions, should never be reduced) now have to be modified.
However, as you know, this is not currently possible, as those parts are blocked from changing the timing.

Is it possible to implement the unlocking of this limitation (always or only upon explicit manual indication)?

Thanks

**Comment by AntonyCorbett** [comment]
@Nickyes Thanks for the feedback. Implemented in v2.2.0.5
